build-repo.sh

```diff
@@ -134,13 +149,15 @@ error=false
 manifests="$(find ${SRCDIR} -name "package.xml") $(find ${extsrc} -name "package.xml")"
 for manifest in ${manifests}; do
   echo +++++++++++++++++++++++++
-  echo ${manifest}
+  echo "manifest: ${manifest}"
   pkgpath=$(dirname ${manifest})
   pkgname=$(basename ${pkgpath})
 
+  echo "latest commit:"
+  git_log_opts="-n1 HEAD ${commit_date_option}"
+  git -C ${pkgpath} log ${git_log_opts}
```

Comment:
Is this line necessary?

Reply:
This is for logging `latest commit:` (L156) information.